### PR TITLE
NG Delegate release notes 786xx

### PR DESCRIPTION
**Pull request by KimberlyFields**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/delegate.md [hunk 1]
```diff
@@ -14,10 +14,28 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
 
 For FirstGen SaaS release notes, see [Harness SaaS Release Notes (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes). For Self-Managed Enterprise Edition, see [Self-Managed Enterprise Edition release notes](/release-notes/self-managed-enterprise-edition).
 
-## February 23, 2023, version 78507
-### Delegate version 78500
+## March 7, 2023, Harness version 786xx, Harness Delegate version 78500
 
-Harness NextGen release 78507 includes the following changes for Harness Delegate.
+Harness NextGen release 786xx includes the following changes for the Harness Delegate.
```
Comment (pratmit) [inline release-notes/delegate.md:19]: We should add the release number (786xx) before publishing

**Review by pratmit — APPROVED** [review]
Approved on the assumption that comment will be resolved.

File: release-notes/delegate.md (hunk 1 above)
Comment (KimberlyFields) [reply release-notes/delegate.md:19]: @pratmit yes, will do. We don't get the final version number until the release is ready to be deployed, so we have to use the xx in the version number as a temporary number.

**Review by douglas-j-bothwell — APPROVED** [review]

**Review by deepakbolangady — APPROVED** [review]
Updated the date and version number in the NG delegate release notes.

**Pull request merged**